.github/workflows/deploy-services.yml

```diff
@@ -27,6 +27,8 @@ on:
 jobs:
   collect-services-versions:
     runs-on: ubuntu-latest
+    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
The documentation recommends the syntax we use here:

![image](https://github.com/Knoblauchpilze/user-service/assets/33599677/0f350165-1635-4f07-a4fc-a2da35a0b7b5)